_posts/2019-02-15-introducing-aquery.md

```diff
@@ -0,0 +1,116 @@
+---
+layout: posts
+title: Introducing Bazel Aquery
+authors:
+ - joeleba
+---
+
+_tl;dr: [`bazel aquery`](https://docs.bazel.build/versions/master/aquery.html) is a new bazel command that queries the action graph, and thus allows you to gain insights about the actions executed in a build (inputs, outputs, command line, …).
+`aquery`’s API is now stable and supported by the Bazel team._
+
+## Why `bazel aquery`?
+
+_"What was the command line that produced this output file?"_
+
+_"Did the new change in my rule implementation affect the current list of actions?"_
```

Comment:
Could you be even more precise about what "current list of actions" means here? For example: the actions generated by the rule? The exact contents of each action generated by the rule? Some list of actions I have saved from a previous query?

Reply:
Done. CYPTAL?